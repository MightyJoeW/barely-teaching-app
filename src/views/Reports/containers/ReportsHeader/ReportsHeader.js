import React from "react";
import { Link } from "react-router-dom";

const ReportsHeader = () => (
  <header className="reports-header">
    <div classsName="content-container">
      <Link className="reports-header__title" to="/reports">
        <h1> Student Behavior Reports </h1>
        <div className="reports-header__actions">
          <Link className="login-button" to="/reports/create">
            Create Report
          </Link>
        </div>
      </Link>
    </div>
  </header>
);

export default ReportsHeader;
