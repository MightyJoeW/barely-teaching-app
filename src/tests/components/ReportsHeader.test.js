import React from "react";
import { shallow } from "enzyme";
import ReportsHeader from "../../views/Reports/containers/ReportsHeader/ReportsHeader";

test("should render ReportsHeader correctly", () => {
  const wrapper = shallow(<ReportsHeader />);
  expect(wrapper).toMatchSnapshot();
});
