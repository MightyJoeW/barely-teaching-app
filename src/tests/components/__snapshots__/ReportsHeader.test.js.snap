// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ReportsHeader correctly 1`] = `
<header>
  <h1>
     Viewing Student Behavior Reports 
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/reports"
  >
    Reports
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/reports/create"
  >
    Create Report
  </NavLink>
</header>
`;
